addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -92,4 +92,80 @@ tour.register('edit_megamenu', {
         run: function () {}, // it's a check
     },
 ]);
+tour.register('edit_megamenu_big_icons_subtitles', {
+    test: true,
+    url: '/?enable_editor=1',
+}, [
+    // Add a megamenu item to the top menu.
+    {
+        content: "Click on a menu item",
+        trigger: '#top_menu .nav-item a',
+    },
+    {
+        content: "Click on 'Link' to open Link Dialog",
+        trigger: '.o_edit_menu_popover a.js_edit_menu',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+        extra_trigger: ".o_web_editor_dialog:visible",
+        trigger: '.modal-body a.js_add_menu[data-type="mega"]',
+    },
+    {
+        content: "Write a label for the new menu item",
+        extra_trigger: ".o_link_dialog",
+        trigger: '.o_link_dialog #o_link_dialog_label_input',
+        run: 'text Megaaaaa!',
+    },
+    {
+        content: "Save the new menu item",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
```

Comment:
Since there is only one primary btn, you can generally omit the "save" part which doesn't bring anything but will fail in versions where we change the save term (from experience this happens)

Reply:
There are two modal-dialog open here so there are multiple `.btn-primary`
![image](https://github.com/odoo/odoo/assets/120459801/0365816c-e3cb-4965-92ff-8f7a06ff1ec8)

Reply:
And FYI even if there is multiple modal opened, adding save wouldn't change anything because they both have save as btn ;-)